roles/keystone/tasks/main.yml

```diff
@@ -80,6 +87,17 @@
   service: name=keystone state=stopped
   when: database_create.changed or force_sync|default('false')|bool
   tags: db-migrate
+  
+- name: disable keystone apache site before db sync
+  apache2_site: name=keystone state=disabled
+  when: database_create.changed or force_sync|default('false')|bool
+  tags: db-migrate
```

Comment:
So right now this code disables the apache front end and stops the uwsgi proc. This will cause HA proxy to throw a 503. IMO i think this is best, as it indicates the back end service is unavailable.